demisto_sdk/commands/common/update_id_set.py

```diff
@@ -2111,14 +2111,14 @@ def find_duplicates(id_set, print_logs, marketplace):
 
     field_list = []
     for field_to_check in field_ids:
-        if has_duplicate(fields, field_to_check, 'Indicator and Incident Fields', print_logs):
+        if has_duplicate(fields, field_to_check, 'Indicator and Incident Fields', print_logs, is_new=True):
             field_list.append(field_to_check)
     lists_to_return.append(field_list)
 
     return lists_to_return
 
 
-def has_duplicate(id_set_subset_list, id_to_check, object_type=None, print_logs=True, external_object=None):
+def has_duplicate(id_set_subset_list, id_to_check, object_type=None, print_logs=True, external_object=None, is_new=False):
```

Comment:
Please change the flag name to `is_create_new` and add an explanation in the doc string